test/paper-item.html

```diff
@@ -45,7 +46,98 @@
       </template>
     </test-fixture>
 
+    <test-fixture id="item-with-input">
+      <template>
+        <div role="list">
+          <paper-item><input></paper-item>
+        </div>
+      </template>
+    </test-fixture>
+
+    <test-fixture id="iconItem-with-input">
+      <template>
+        <div role="list">
+          <paper-icon-item><input></paper-icon-item>
+        </div>
+      </template>
+    </test-fixture>
+
     <script>
+      suite('paper-item basic', function() {
+        var item, clickHandler;
+
+        setup(function() {
+          item = fixture('item').querySelector('paper-item');
+          clickHandler = sinon.spy();
+          item.addEventListener('click', clickHandler);
+        });
+
+        test('space triggers a click event', function() {
+          MockInteractions.pressSpace(item);
+          setTimeout(function(){
+            expect(clickHandler.callCount).to.be.equal(1);
+          }, 100);
```

Comment:
I left this comment in another PR - would it be okay to be more explicit about how much time we should wait for this?